packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -1605,7 +1606,7 @@ describe('Collection aggregations tab', function () {
       );
 
       await browser.selectOption(
-        Selectors.AggregationWizardSortFormDirectionSelect(0),
+        `${Selectors.AggregationWizardSortFormDirectionSelect(0)} button`,
```

Comment:
I have no recollection of it. I think I moved it around while in Berlin but can't remember if it was really required or not. I think the reasoning is that the selector points to the component and the button is a thing inside there and therefore by not making it point at the button the selector is more reusable.

All I know is that this has been rock solid ever since, so probably fine ;)